File: fibonacci.java
public class fibonacci {
    public static void Fibo(int n) {
        int a = 0;
        int b = 1;
        int c;
        System.out.print(a + ", " + b);
        for (int i = 2; i < n; i++) {
            c = a + b;
            System.out.print(", " + c);
            a = b;
            b = c;
        }
        System.out.println();
    }
}

File: fibonacci.js
function fibonacci(n) {
    if (n <= 1) {
      return n;
    } else {
      return fibonacci(n - 1) + fibonacci(n - 2);
    }
  }
  var n = prompt("Digite um número: ");
  var fib = fibonacci(n);
  console.log("O " + n + " na Fibonacci é: " + fib);